packages/vite/src/node/server/index.ts

```diff
@@ -215,7 +215,7 @@ export interface FileSystemServeOptions {
    * This will have higher priority than `allow`.
    * picomatch patterns are supported.
    *
-   * @default ['.env', '.env.*', '*.crt', '*.pem']
+   * @default ['.env', '.env.*', '*.{crt,pem}', '**'+'/.git/**']
```

Comment:
`**/` needs to be escaped so that it doesn't end the comment. `**\/` showed up as-is in vscode.
![image](https://github.com/user-attachments/assets/6d46cd31-b3f4-4827-9fcb-dde633ef3aa1)
I went with `'**'+'/.git/**'` as `**\/` has a different meaning (it's a bit messy though).

Reply:
Using `\/` feels better I think, it's escaping in the string so I think the behaviour should be the same? Using a `+` makes it a bit longer.

Reply:
> it's escaping in the string so I think the behaviour should be the same?

Ah, that's true, will update 😅

Reply:
```suggestion
   * @default ['.env', '.env.*', '*.{crt,pem}', '**\/.git/**']
```